internal/http/handler/api_server_test.go

```
@@ -77,6 +78,59 @@ func TestToRoomResponseAllowsNilOptionalFields(t *testing.T) {
+				CollectedRent:    40000,
+				OverdueBillCount: 2,
+			},
+			RecentJournals: []appproperty.DashboardRecentJournal{
+				{ID: "60000000-0000-0000-0000-000000000001", Type: "journal_log", Content: "Changed lobby light", CreatedAt: createdAt},
+			},
+		},
+	}
+	server := &APIServer{propertyDashboard: appproperty.NewDashboardService(repo)}
+	recorder := httptest.NewRecorder()
+	c, _ := gin.CreateTestContext(recorder)
+	c.Request = httptest.NewRequest(http.MethodGet, "/api/v1/properties/"+propertyID+"/dashboard", nil)
```

golangci-lint (2.11.4)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)